src/planner.c

```diff
@@ -1043,6 +1043,8 @@ timescaledb_set_rel_pathlist(PlannerInfo *root, RelOptInfo *rel, Index rti, Rang
 	{
 		case TS_REL_HYPERTABLE_CHILD:
 			/* empty child is not of interest */
+			if (root->parse->commandType == CMD_DELETE)
```

Comment:
Should we mark it as empty for updates as well? Also I think the entire PR description should go as a comment here.

Reply:
Yes we should treat update the same, but i got test failures when doing this for updates, so i will do it in separate step.